specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/examples/CreateEnterpriseChannel.json

```diff
@@ -0,0 +1,77 @@
+{
+  "parameters": {
+    "subscriptionId": "subscription-id",
+    "resourceGroupName": "OneResourceGroupName",
+    "api-version": "2018-07-12",
+    "resourceName": "contoso-dl",
+    "parameters": {
+      "location": "West US",
```

Comment:
So, do enterprise channels have a location? Bots and channels are global. Seems accurate since there will be an app service for them, but want to confirm.

Reply:
Enterprise Channels have a location - there are resources associated with just the channel itself and they are deployed to this region.